packages/components/src/components/domainproperties/issues/IssuesPropertiesPanel.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import { Form } from 'react-bootstrap';
+
+import { Utils } from '@labkey/api';
+
+import { List } from 'immutable';
+
+import { DomainDesign } from '../models';
+
+import { BasePropertiesPanel, BasePropertiesPanelProps } from '../BasePropertiesPanel';
+import {
+    InjectedDomainPropertiesPanelCollapseProps,
+    withDomainPropertiesPanelCollapse,
+} from '../DomainPropertiesPanelCollapse';
+import {getCoreGroups, getCoreUsersInGroups} from '../../permissions/actions';
+import { Principal } from '../../..';
+
+import { AssignmentOptions, IssuesBasicPropertiesFields } from './IssuesPropertiesPanelFormElements';
+import { IssuesModel } from './models';
+import {UserGroup} from "../../permissions/models";
+
+const PROPERTIES_HEADER_ID = 'issues-properties-hdr';
+
+interface OwnProps {
+    model: IssuesModel;
+    onChange: (model: IssuesModel) => void;
+    successBsStyle?: string;
+}
+
+type Props = OwnProps & BasePropertiesPanelProps;
+
+interface State {
+    isValid: boolean;
+    coreGroups: List<Principal>;
+    coreUsers: List<UserGroup>;
+}
+
+export class IssuesPropertiesPanelImpl extends React.PureComponent<
+    Props & InjectedDomainPropertiesPanelCollapseProps,
+    State
+> {
+    constructor(props: Props & InjectedDomainPropertiesPanelCollapseProps) {
+        super(props);
+
+        this.state = {
+            isValid: true,
+            coreGroups: List<Principal>(),
+            coreUsers: List<UserGroup>()
```

Comment:
These are now moved to AssignmentOptions component in IssuesListDefPropertiesPanelFormElements.tsx -- i had to initialize them otherwise I am getting an error during load on getFilteredCoreUsers() --> coreUsers.filter (~line 255).

Reply:
Hmm, I see. Another option would be to initialize them as `undefined` and then in that input component that uses them to get a filtered list, instead render a `<LoadingSpinner>` when undefined instead of the `<SelectInput>`

Reply:
Ah good to know! That worked out great!